docker/docker_image_src.go

```diff
@@ -55,6 +55,15 @@ func (s *dockerImageSource) IntendedDockerReference() string {
 	return fmt.Sprintf("%s:%s", s.ref.Name(), s.tag)
 }
 
+// ManifestMIMETypes returns a slice of supported MIME types
+func manifestMIMETypes() []string {
```

Comment:
What does “supported” mean here, exactly, considering that all callers in #73 only use V2S1?

Maybe we just don’t need this function?